### ISSUE TITLE
Converting Vanilla2-Board

### ISSUE THREAD

**Issue by ToeiRei** [mybb/merge-system#194]
Hi guys,

I tried to migrate from Vanilla2 and run into the following issue:

Vanilla table names:    GDN_Role
What merger expects:    GDN_role

My solution was hacking the group names (caps) in the merger files as I didn't want to alter the live system of the vanilla (to be merged). Looks like Vanilla 2.2 got the names in CamelCase...

**Comment by ToeiRei** [comment]
After changing the merger a bit, I could get things to work. Patch here:
https://git.stargazer.at/vbauer/mybb-merger/commit/192156d69069963e141f44ef2cecd2810f213599.patch

**Comment by JN-Jones** [comment]
This has been reported before IIRC. Still need to check how Vanilla handles there upgrades and whether it's save to simply change it or whether we need to test for it.

**Comment by luisalvarado** [comment]
Hi and thank you for the work here. I tried the latest vanilla and yes, I too needed to do a lot of custom fixes inside the SQL file before the import worked. I am not the owner of the vanilla data, am just the developer that is migrating 2 sites (2 different owners) because I recommended mybb to them. Currently it is working more or less correct after the migration but it would be awesome if the whole merge-system was not left in the dark. the MyBB team has created an amazing tool where, people from other platforms I have seen... let me rephrase that, from other REALLY horrible platforms I have seen, can be migrated to mybb and they can, for many cases, enjoy a damn forum that is well made, maintained and has put a lot of thought into making it correctly from the get go. The merger system is simply icing on the mybb cake and it would be sad to have it go away. Just my feedback from someone that has migrated a lot of business owners to your forum and have never heard one complaint they did not like it. Thank you for the amazing work.
